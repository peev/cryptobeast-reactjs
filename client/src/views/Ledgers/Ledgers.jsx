import React from 'react';
// import PropTypes from 'prop-types';
import withStyles from '@material-ui/core/styles/withStyles';
import Grid from '@material-ui/core/Grid';

import { ItemGrid } from './../../components';

import iconsStyle from './../../variables/styles/iconsStyle';

function Ledgers() {
  return (
    <Grid container>
      <ItemGrid xs={12} sm={12} md={12}>
        <p>Ledgers is under construction...</p>
      </ItemGrid>
    </Grid>
  );
}

// Ledgers.propTypes = {
//   classes: PropTypes.object.isRequired,
// };

export default withStyles(iconsStyle)(Ledgers);
