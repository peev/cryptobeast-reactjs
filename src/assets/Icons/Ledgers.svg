<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33.43 30.12"><defs><style>.cls-1{fill:none;stroke:#fff;stroke-miterlimit:10;}.cls-2{fill:#fff;}</style></defs><title>Asset 5</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-1" x="0.5" y="0.89" width="6.1" height="28.55"/><rect class="cls-1" x="10.3" y="0.89" width="6.1" height="28.55"/><rect class="cls-1" x="21.74" y="0.79" width="6.1" height="28.55" transform="translate(-3.77 9.96) rotate(-21.16)"/><line class="cls-1" x1="0.57" y1="21.52" x2="6.67" y2="21.52"/><line class="cls-1" x1="0.5" y1="25.29" x2="6.6" y2="25.29"/><line class="cls-1" x1="10.44" y1="21.52" x2="16.55" y2="21.52"/><line class="cls-1" x1="10.3" y1="25.29" x2="16.4" y2="25.29"/><line class="cls-1" x1="24.69" y1="22.62" x2="30.38" y2="20.41"/><line class="cls-1" x1="25.72" y1="26.32" x2="31.41" y2="24.12"/><circle class="cls-2" cx="3.55" cy="4.3" r="1.39"/><circle class="cls-2" cx="13.35" cy="4.3" r="1.39"/><circle class="cls-2" cx="20.78" cy="4.7" r="1.39"/></g></g></svg>