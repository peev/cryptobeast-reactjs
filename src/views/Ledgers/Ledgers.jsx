import React from 'react';
import PropTypes from 'prop-types';
import { withStyles, Grid } from 'material-ui';

import { ItemGrid } from 'components';

import iconsStyle from 'variables/styles/iconsStyle';

function Ledgers({ ...props }) {
  return (
    <Grid container>
      <ItemGrid xs={12} sm={12} md={12}>
        <p>Ledgers is under construction...</p>
      </ItemGrid>
    </Grid>
  );
}

Ledgers.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(iconsStyle)(Ledgers);
