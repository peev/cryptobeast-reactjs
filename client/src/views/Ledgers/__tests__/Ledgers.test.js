import React from 'react';
import { shallow } from 'enzyme';
import Ledgers from './../Ledgers';

describe('Views/Ledgers.jsx', () => {
  const ledgers = shallow(<Ledgers />);

  it('should render without crashing', () => {
    expect(ledgers).toMatchSnapshot();
  });
});
