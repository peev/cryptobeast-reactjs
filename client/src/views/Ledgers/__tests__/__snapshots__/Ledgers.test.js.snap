// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views/Ledgers.jsx should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Ledgers) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "iframe": "Ledgers-iframe-01",
        "iframeContainer": "Ledgers-iframeContainer-02",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "iframe": "Ledgers-iframe-01",
          "iframeContainer": "Ledgers-iframeContainer-02",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
